pkg/cluster/nodes/node.go

```diff
@@ -163,10 +163,9 @@ func tryUntil(until time.Time, try func() bool) bool {
 func (n *Node) LoadImages() {
 	// load images cached on the node into docker
 	if err := n.Command(
-		"find",
-		"/kind/images",
-		"-name", "*.tar",
-		"-exec", "docker", "load", "-i", "{}", ";",
+		"/bin/bash", "-c",
+		// use xargs to load images in parallel
+		`find /kind/images -name *.tar -print0 | xargs -0 -n 1 -P $(nproc) docker load -i`,
```

Comment:
/this-is-fine

Reply:
yeah.

So this means that we list everything, collect it up in xargs, and then run one docker load per core.

It's not actually terribly important how many cores, mostly that we run multiple in parallel because that's definitely faster, non-scientific experimentation suggested nproc is good enough for now.

Reply:
FWIW this is actually pretty simple:

`find`: we're using find to locate files
`/kind/images`: in this directory
`-name *.tar`: named `*.tar`
`-print0`: print the names out null terminated, so we can support files with spaces etc properly
`| xargs`: pipe the results to xargs for running over each file
`-0`: tell xargs to expect null terminated inputs
`-n 1`: tell xargs to run each input individually (IE one file at a time, since `docker load` requires this)
`-P $(nproc)`: tell xargs to run in parallel with at most one process per core
`docker load -i`: xargs should run `docker load -i ${input}` to load the image, `-i` tells docker load to use a file name instead of stdin.

Reply:
would be nice to have this as a comment above the function.
but also why not call `docker load...` in go routines. would that work?

Reply:
that's slower and more code. if someone can't read this snippet they shouldn't touch it :^)

Reply:
to be clear: there's always been a comment here about what we're doing, before anyone went to edit this I'd expect them to understand what it does and why. It shouldn't be a big deal to check the docs for these commands and the only bash syntax we use here is a pipe.